test/interop/fortran/multidimArray/chapelProcs.chpl

```diff
@@ -0,0 +1,53 @@
+use ISO_Fortran_binding;
+
+proc toChapelArray(ref A: CFI_cdesc_t, param rank = 1) {
```

Comment:
Maybe I'm missing something but I don't see where this function gets used.  Does it?

Reply:
It's not used and wasn't complete.  I removed it.